### ISSUE TITLE
'Error fetching accounting' 'No oracles used' after executing DLC

### ISSUE THREAD

**Issue by user411** [bitcoin-s/bitcoin-s#3826]
**Describe the bug**
I created a numeric contract in the web UI and accepted it through Tor in another web UI. I later executed the contract with oracle signatures from the first web UI. The second wallet now does not seem to be able to locate any DLCs in the wallet.

**To Reproduce**
1. Generate a new numeric offer in the web UI on machine 1
2. Accept offer via Tor on web UI on machine 2
3. Execute DLC on machine 1
4. Check state of DLCs on machine 2

**Expected behavior**
No error in the second wallet.

**Logs**
```
[info] 2021-11-15T15:47:43UTC ERROR [WalletGUIModel] Error fetching accounting
[info] java.lang.RuntimeException: Request failed: requirement failed: Error, no oracles used
[info] 	at org.bitcoins.cli.ConsoleCli$.error$1(ConsoleCli.scala:1703)
[info] 	at org.bitcoins.cli.ConsoleCli$.$anonfun$exec$1(ConsoleCli.scala:2075)
[info] 	at scala.util.Try$.apply(Try.scala:210)
[info] 	at org.bitcoins.cli.ConsoleCli$.exec(ConsoleCli.scala:2017)
[info] 	at org.bitcoins.gui.WalletGUIModel.org$bitcoins$gui$WalletGUIModel$$updateWalletAccounting(WalletGUIModel.scala:168)
[info] 	at org.bitcoins.gui.WalletGUIModel$UpdateWalletInfoRunnable$.$anonfun$run$1(WalletGUIModel.scala:38)
[info] 	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.scala:18)
[info] 	at scalafx.application.Platform$$anon$1.run(Platform.scala:148)
[info] 	at com.sun.javafx.application.PlatformImpl.lambda$runLater$10(PlatformImpl.java:447)
[info] 	at java.base/java.security.AccessController.doPrivileged(Native Method)
[info] 	at com.sun.javafx.application.PlatformImpl.lambda$runLater$11(PlatformImpl.java:446)
[info] 	at com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:96)
[info] 	at com.sun.glass.ui.gtk.GtkApplication._runLoop(Native Method)
[info] 	at com.sun.glass.ui.gtk.GtkApplication.lambda$runLoop$11(GtkApplication.java:277)
[info] 	at java.base/java.lang.Thread.run(Thread.java:829)
[info] java.lang.IllegalArgumentException: requirement failed: Error, no oracles used
[info] 	at scala.Predef$.require(Predef.scala:337)
[info] 	at org.bitcoins.dlc.wallet.util.DLCStatusBuilder$.getOracleOutcomeAndSigs(DLCStatusBuilder.scala:221)
```

**Comment by user411** [comment]
Tested this with executing on the second machine. The first machine has the same behavior as the second had previously.

**Comment by user411** [comment]
May be impacted by #3829, will test.

**Comment by user411** [comment]
This issue was fixed by #3829
